components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityTenantUtil.java

```diff
@@ -328,9 +328,7 @@ public static int getTenantIdOfUser(String username) throws IdentityRuntimeExcep
      */
     public static String getTenantDomainFromContext() {
 
-        String tenantDomain = (String) IdentityUtil.threadLocalProperties.get().get
-                (IdentityCoreConstants.TENANT_NAME_FROM_CONTEXT);
-        return tenantDomain;
+        return (String) IdentityUtil.threadLocalProperties.get().get(IdentityCoreConstants.TENANT_NAME_FROM_CONTEXT);
```

Comment:
Can there be a possibility for 'IdentityUtil.threadLocalProperties.get()' to be null

Reply:
Looks like it cannot be null since it is initialized with an initial value of an empty map in
https://github.com/wso2/carbon-identity-framework/blob/79cb2c3512542d3d75f3572b662f81ea7041eb32/components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java#L94-L100